### PR TITLE
OKTA-798706 ref landing page update

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?**  Updates to reference landing page
- **Is this PR related to a Monolith release?**  None

### Resolves:

* [OKTA-798706](https://oktainc.atlassian.net/browse/OKTA-798706)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/index.md](https://github.com/okta/okta-developer-docs/blob/c793131fb3058c6474a8b3b16bf1b1755dd29bea/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c72016a6-297c-4dbc-a6b4-eacef15fa719) | 52  | 62  | 100  | 87  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.86cf7c9b-5aaf-41f1-a96c-7cee22107685?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/index.md](https://github.com/okta/okta-developer-docs/blob/413f26917532723fbe733beaa66cc14215affae6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/08f9b92f-3af6-4780-808a-89eecd8f6825) | 62  | 62  | 100  | 87  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.7fcc89ab-0c9b-49f1-838d-75b9c8dda6e5?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/reference/index.md [hunk 1]
```diff
@@ -6,28 +6,58 @@ title: Reference overview
 
 # Reference overview
 
-Details on parameters, requests, and responses for Okta API endpoints.
+This section provides reference information related to the [Okta API](https://developer.okta.com/docs/api/) endpoints.
 
-## Core Okta API
+## Classic Engine Content
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/index.md:11]:
```suggestion
## Classic Engine content
```

File: packages/@okta/vuepress-site/docs/reference/index.md [hunk 2]
```diff
@@ -6,28 +6,58 @@ title: Reference overview
...
 
-The Core Okta API is the primary way that apps and services interact with Okta. You can use it to implement basic auth functions such as signing in your users and programmatically managing your Okta objects.
+These references are specific to Classic Engine only.
 
-[Explore the core Okta API](/docs/reference/core-okta-api/)
+[Learn more](/reference/classic-engine/)
 
-<Cards><Card href="/docs/reference/postman-collections/" cardTitle="Postman Collections" :showFooter=true>Import any Okta API collection for Postman.</Card><Card href="/docs/reference/api/asa/introduction/" cardTitle="Advanced Server Access API" :showFooter=true>Scale your control of servers with automation.</Card></Cards>
+## Error Codes
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/index.md:17]:
```suggestion
## Error codes
```

File: packages/@okta/vuepress-site/docs/reference/index.md [hunk 3]
```diff
@@ -6,28 +6,58 @@ title: Reference overview
...
 
-## Okta hooks
+This document contains a complete list of all errors that the Okta API returns.
 
-Okta event and inline hooks allow you to integrate custom functionality into specific Okta process flows. Event hooks send Okta events of interest to your systems as they occur, just like a webhook. Inline hooks allow developers to modify in-flight Okta processes with custom logic and data from a non-Okta source. For example, you can migrate users from another data store and keep the user’s current password with a [password inline hook](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/InlineHook/#tag/InlineHook/operation/createPasswordImportInlineHook).
+[Learn more](/docs/reference/error-codes/)
+
+## Event Types
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/index.md:23]:
```suggestion
## Event types
```

File: packages/@okta/vuepress-site/docs/reference/index.md [hunk 4]
```diff
@@ -6,28 +6,58 @@ title: Reference overview
...
+
+Event types are the primary method of categorization within the Okta eventing platform. They allow consumers to easily group notable system occurrences based on behavior. This resource contains the complete event type catalog of this platform.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/index.md:25]:
```suggestion
Event types are the primary method of categorization within the Okta events platform. They allow consumers to easily group notable system occurrences based on behavior. This resource contains the complete event type catalog of this platform.
```

File: packages/@okta/vuepress-site/docs/reference/index.md [hunk 5]
```diff
@@ -6,28 +6,58 @@ title: Reference overview
...
 [Learn more](/docs/reference/okta-expression-language/)
 
-## System for Cross-domain Identity Management
+## Expression Language in Identity Engine
+
+This Expression Language reference is specific to Identity Engine.
+
+[Learn more](/docs/reference/okta-expression-language-in-identity-engine/)
+
+## Rate Limits
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/index.md:41]:
```suggestion
## Rate limits
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/reference/index.md](https://github.com/okta/okta-developer-docs/blob/ebab5ef798f0949789220f5d16e6065b87289124/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Freference%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7ae7942e-2724-46c4-9c43-db43c3220285) | 52  | 62  | 100  | 87  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.234db9f6-fbed-4aa4-84f5-9c964b439f55?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**